### ISSUE TITLE
Line length

### ISSUE THREAD

**Issue by rggjan** [Jintin/Swimat#188]
**Feature request**
Make it possible to define a maximum line length, and automatically wrap lines which are too long, or combine short enough lines, similar to what clang-format does for objective-c and c++ code.

**Comment by adozenlines** [comment]
Combining lines under swift will mean the use of semi colons, is that something you really want to do?

**Comment by rggjan** [comment]
No! I was talking about combining and splitting single lines without semicolon. For example changing this:

assert(age >= 0,
             "A person's age can't be less than zero.")

To this:
assert(age >= 0, "A person's age can't be less than zero.")

Or the other was around, depending on preferred maximum line length.

**Comment by jasjuang** [comment]
+1, it will be great to have this feature!
